File: find_toargb32.py
import json, os, subprocess

with open('.dart_tool/package_config.json', 'r') as f:
    packages = json.load(f)['packages']

for pkg in packages:
    uri = pkg['rootUri']
    if uri.startswith('file:///'):
        path = os.path.normpath(uri[8:])
        cmd = f'findstr /S /M "toARGB32" "{path}\\*.dart"'
        try:
            res = subprocess.run(cmd, shell=True, capture_output=True, text=True)
            if res.stdout:
                print(f'FOUND in {pkg["name"]}:\n{res.stdout.strip()}')
        except Exception as e:
            pass

File: find_toargb32.dart
// ignore_for_file: avoid_print
import 'dart:convert';
import 'dart:io';

void main() {
  final configStr = File('.dart_tool/package_config.json').readAsStringSync();
  final config = jsonDecode(configStr);
  final packages = config['packages'] as List;

  for (var pkg in packages) {
    final rootUri = pkg['rootUri'] as String;
    if (rootUri.startsWith('file:///')) {
      final path = Uri.parse(rootUri).toFilePath();
      final dir = Directory(path);
      if (dir.existsSync()) {
        for (var entity in dir.listSync(recursive: true)) {
          if (entity is File && entity.path.endsWith('.dart')) {
            try {
              final content = entity.readAsStringSync();
              if (content.contains('toARGB32')) {
                print('FOUND in ${pkg["name"]}: ${entity.path}');
              }
            } catch (e) {
              // ignore
            }
          }
        }
      }
    }
  }
}
